app/controllers/home_controller.rb

```diff
@@ -1,11 +1,18 @@
 class HomeController < ApplicationController
   def index
-    if super_admin?
-      redirect_to super_admin_organisations_path
-    elsif new_super_admin?
-      redirect_to super_admin_neo_dashboard_path
-    else
-      redirect_to(current_organisation.ips.empty? ? new_organisation_setup_instructions_path : overview_index_path)
-    end
+    destination =
+      if current_user.super_admin?
+        if current_user.new_super_admin?
+          super_admin_neo_dashboard_path
+        else
```

Comment:
I am not sure we need this else but let's check afterwards

Reply:
as per our call, this was needed for routing purposes 🔀